package.json

```diff
@@ -73,21 +79,21 @@
   },
   "dependencies": {
     "@kiwicom/orbit-components": "^0.0.0-rc1",
-    "apollo-boost": "0.1.4",
     "axios": "^0.18.0",
-    "graphql": "0.13.2",
     "babel-runtime": "^6.26.0",
     "i18next": "10.5.0",
+    "isomorphic-fetch": "2.2.1",
```

Comment:
Wouldn't `axios` be enough?

Reply:
`axios` is actually adding additional functionality which we don't need - transformers, interception or cancellation

Reply:
Agree, then we should replace all `axios` uses with polyfilled `fetch`. I'll raise  an issue for that